docs/admin/cluster-large.md

```diff
@@ -36,7 +36,7 @@ Documentation for other releases can be found at
 
 ## Support
 
-At v1.1, Kubernetes supports clusters up to 250 nodes with 30 pods per node and 1-2 containers per pod.
+At v1.2, Kubernetes supports clusters up to 1000 nodes with 30 pods per node and 1-2 containers per pod.
```

Comment:
Yes, I will send pr to do that change shortly. @davidopp could you please update this doc to reflect that change too? Thanks!